packages/material-ui/src/Portal/Portal.js

```diff
@@ -34,10 +34,14 @@ class Portal extends React.Component {
     ) {
       this.setMountNode(this.props.container);
 
-      // Only rerender if needed
-      if (!this.props.disablePortal) {
-        this.forceUpdate(this.props.onRendered);
-      }
+      // This might be triggered earlier than the componentDidUpdate of a child
+      // We need to account for this.
+      setTimeout(() => {
+        // Only rerender if needed
+        if (!this.props.disablePortal) {
+          this.forceUpdate(this.props.onRendered);
+        }
+      }, 0);
```

Comment:
0 is implicit when not provided.

Reply:
Should this be changed as well? https://github.com/mui-org/material-ui/blob/7c29d28ac9ec1cbd5684069f949b00c95dd34fa4/packages/material-ui/src/Tooltip/Tooltip.js#L131-L136